client/app/views/steps/password.coffee

```diff
@@ -22,6 +22,11 @@ module.exports = class PasswordView extends StepView
         @$visibilityButton = @$('[action=password-visibility]')
         @$visibilityIcon = @$('.icon use')
         @$strengthBar = @$('progress')
+        @$errorContainer=@$('.errors')
+        if @error
+            @handleErrorMessage(@error)
```

Comment:
About the naming here: I am not a big fan of having a call to a method named `handleSomething` when we are not in the situation of an event handling. I suggest:
```Coffeescript
if @error
    @renderError(@error)
// or
if @error
    showError(@error)
```
and a little below:
```Coffeescript
handleErrorMessage: (error) ->
    showError(error)
```

What do you think ?